src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -657,11 +672,11 @@
 
 "App_Information_Contact_Hotline_Text" = "+49 (0)800 7540001";
 
-"App_Information_Contact_Hotline_Number" = "+49 (0)800 7540001";
+"App_Information_Contact_Hotline_Number" = "+498007540001";
 
 "App_Information_Contact_Hotline_Description" = "Our technical customer service is there for you. Waiting time is currently less than 5 minutes.";
 
-"App_Information_Contact_Hotline_Terms" = "Business hours:\nMonday to Friday: 8am - 10pm\nSaturday and Sunday: 10am - 10pm\nThe standard charges of your telephone provider apply.";
+"App_Information_Contact_Hotline_Terms" = "Business hours:\nMonday to Saturday: 8am - 10pm\n(except national holidays)";
```

Comment:
why is this not abbreviated like in the German version?

Reply:
Please ask translation folks